src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx

```diff
@@ -6,29 +6,25 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-As part of New Relic's support for OpenTelemetry Protocol (OTLP) attributes can contain arrays. Arrays are a data structure that contains an ordered collection of values.
+As part of our support for OpenTelemetry Protocol (OTLP), attributes can contain arrays within New Relic. But what are arrays? Essentially, arrays are data structures containing an ordered collection of values that you can query with NRQL. Typically, arrays display in a format like `[1, 2, 3, 4, 5]` or `["US", "CA", "UK"]`.
 
-Currently only simple arrays are supported. You can query them with NRQL. To quickly verify that any data is in an array, you can use the JSON chart type to see if the data displays as an array.
-
-## Simple arrays [#simple-arrays]
-
-Simple arrays are used to store a list of values of the same primitive type; for example, numeric, boolean, or string. An array looks like `[1, 2, 3, 4, 5]` or `["US", "CA", "UK"]`.
+Currently, we only support simple arrays. You can use simple arrays to store a list of values of the same primitive type; for example, numeric, boolean, or string. To quickly verify that an array contains any data, you can use the JSON chart type to see if the data displays in an array structure.
```

Comment:
Contra to my comment above, I think it's important to define what a simple arrays is vs more complex arrays.

Do you just mean that it's a single dimensional array and we don't support multi-dimensional ones?